File: library/src/main/java/org/coner/crispy_fish/filetype/staging/StagingFilenames.java
package org.coner.crispy_fish.filetype.staging;

import java.util.regex.Pattern;

public class StagingFilenames {

    static final Pattern ORIGINAL_FILE_PATTERN = Pattern.compile(".*.st1$");
    static final Pattern REACCEPTED_FILE_PATTERN = Pattern.compile(".*.st1.\\d{3}$");

    static String getReacceptedFileBaseName(String originalFileBaseName) {
        return originalFileBaseName + "_st1";
    }
}

File: library/src/main/kotlin/tech/coner/crispyfish/filetype/staging/StagingFilenames.kt
package tech.coner.crispyfish.filetype.staging

import java.util.regex.Pattern

object StagingFilenames {

    const val ORIGINAL_FILE_DAY_1_EXTENSION = "st1"
    const val ORIGINAL_FILE_DAY_2_EXTENSION = "st2"
    val ORIGINAL_FILE_DAY_1: Pattern = Pattern.compile(".*.$ORIGINAL_FILE_DAY_1_EXTENSION")
    val ORIGINAL_FILE_DAY_2: Pattern = Pattern.compile(".*.$ORIGINAL_FILE_DAY_2_EXTENSION")
}
